src/main/java/domain/Car.java

```diff
@@ -0,0 +1,30 @@
+package domain;
+
+public class Car {
+
+    private String carName;
+    private int position;
+
+    public Car(String carName) {
+        this.carName = carName;
+        this.position = 0;
+    }
+
+    public Car(String carName, int position) {
+        this.carName = carName;
+        this.position = position;
+    }
+
+    public void forward(){
+        this.position++;
+    }
```

Comment:
position의 값을 1개 올려서 한칸 앞으로 전진하는 차의 역할을 표현한 것이 정말 좋네요!

forward라는 메서드 명에 대해서 어떻게 생각하시나요?
보통 메서드명은 동사로 사용하는 게 좋다고 하는데요.

메서드는 여러 일을 하는데요.
- 객체의 상태를 바꾸고 
- (상태에 따른)검증같은 판단을 하고
- ...

메서드는 Class의 명세를 나타낸다고 생각합니다.
그래서 메서드명이 `역할`을 더 잘 보여주면 어떨까요?
```suggestion
    public void moveForward(){
        this.position++;
    }
```

Reply:
앞으로는 메서드명 지을 때 좀 더 신중하게 생각해서 짓겠습니다! moveForward 좋은 메서드명 같아요. 반영했습니다 ~!